app/AppKernel.php

```diff
@@ -34,4 +34,13 @@ public function registerBundles()
 
         return array_merge(parent::registerBundles(), $bundles);
     }
+
+    protected function getContainerBaseClass()
```

Comment:
However this change should be reverted. This change has been applied to [TestKernel](https://github.com/Sylius/Sylius/blob/master/app/TestKernel.php#L69) already.